generatedocs/README.md

```diff
@@ -28,6 +43,8 @@ go run ./generatedocs/compile/compile.go
 
 ### Generating markdown files
 
+To generate a new `settingup.md` only use the following command:
```

Comment:
Suggestion:
"To only generate a new `settingup.md` file, use the following command:"